### ISSUE TITLE
Setting to always embed fonts when saving to svg, (*.drawio.svg)

### ISSUE THREAD

**Issue by mawid6** [jgraph/drawio#2886]
* [x] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

**Is your feature request related to a problem? Please describe.**
I like to **_save_** and work with diagrams in png or svg format with the diagram embedded, e g `foobar.drawio.svg`. When using open fonts like Fira Sans, Roboto, etc, the fonts are not embedded in the created `svg`. 
Contrast with `svg` **_export_** which has the option to embed fonts (implemented "in" #1702).

**Describe the solution you'd like**
Ability to set that fonts are always embedded when **_saving_** to `svg` format. A flag/checkbox, in the save dialogue, that perhaps persists with the diagram to make it flexible... 
Could be an application setting also, but that would be less flexible.

**Describe alternatives you've considered**
Workaround right now is to stop using custom/ open fonts (when using svg embedded files as storage) or not use svg, but go for `.drawio.png` files instead.

**Additional context**
Also related: [#404](https://github.com/jgraph/drawio-desktop/issues/484) (which in turn is maybe actually a dupe of #1702).

**Comment by mawid6** [comment]
Hi! It seems to me this is potentially an easy fix: This is already works for export to svg. 

If you do export to svg, the fonts are embedded.

Why not reuse that code / functionality in the save as svg? 
Just a note that this **could** be an easy fix.


This is a showstopper for using anything but helvetica and times new roman, etc.

-> ...when saving embedded, i e  as .drawio.svg. 

So drawio svg:s in any git repo will not display correctly if they use any but the most common cross platform fonts.

**Comment by penyuan** [comment]
Is there an update on this issue? Being able to embed fonts will be so useful.

**Comment by davidjgraph** [comment]
No, there is clearly no update on this issue, you can see that from there being no update...

**Comment by davidjgraph** [comment]
Please check 24.0.6 to see if that completes this issue.

**Comment by djuran** [comment]
v26.0.6 does still *not* fix the issue

**Comment by rhumie** [comment]
v26.0.15 does still not fix the issue.
